### PR TITLE
odometry: restructure to include kick support state

### PR DESCRIPTION
**Pull request by SammyRamone**
It is necessary to also include the kick support state into the odometry. This was previously missing.

## Necessary checks
- [x] Update package version
- [x] Run linters
- [x] Run `catkin build`
- [x] Write documentation
- [x] Test on your machine
- [ ] Test on the robot

### CODE REVIEW

File: bitbots_odometry/launch/odometry.launch [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <launch>
+    <node name="motion_odometry" pkg="bitbots_odometry" type="motion_odometry" />
```
Comment (NFiedler) [inline bitbots_odometry/launch/odometry.launch]: I'm a bit concerned about this... do we have to remove /rename the walk odometry in other packages?

Comment (SammyRamone) [reply bitbots_odometry/launch/odometry.launch]: yes we have to always launch the new odometry.launch file now.
I think it make semantically more sense to have one odometry launch file which launches all nodes necessary for the odometry

**Comment by timonegk** [comment]
The kick is sometimes in the wrong frame

**Comment by SammyRamone** [comment]
> The kick is sometimes in the wrong frame

should be fixed

File: bitbots_dynamic_kick/src/kick_node.cpp [hunk 2]
```diff
@@ -22,7 +22,7 @@ KickNode::KickNode() :
   ik_.init(kinematic_model);
 
   joint_goal_publisher_ = node_handle_.advertise<bitbots_msgs::JointCommand>("kick_motor_goals", 1);
-  support_foot_publisher_ = node_handle_.advertise<std_msgs::Char>("dynamic_kick_support_state", 1);
+  support_foot_publisher_ = node_handle_.advertise<std_msgs::Char>("dynamic_kick_support_state", 1, true);
```
Comment (jgueldenstein) [inline bitbots_dynamic_kick/src/kick_node.cpp]: maybe add latch=true instead of just true

Comment (jgueldenstein) [reply bitbots_dynamic_kick/src/kick_node.cpp]: it seems this is a python thing to do, i have added it as a comment for clarification

**Pull request merged**